README.md

```diff
@@ -1,7 +1,150 @@
+
+To install the backend, follow the steps below:
+
+### Gateway Installation
+
+1.  If you do not have the Common Gateway installed, you can follow the installation guide provided [here](https://github.com/ConductionNL/commonground-gateway#readme). The Common Gateway installation is required for the backend setup. You can choose any installation method for the gateway, such as Haven, Kubernetes, Linux, or Azure, and any database option like MySQL, PostgreSQL, Oracle, or MsSQL. The gateway framework handles this abstraction.
+
+### XxllncZGWBundle Installation - Admin-UI
+
+1.  After successfully installing the Gateway, access the admin-ui and log in.
+2.  In the left menu, navigate to "Plugins" to view a list of installed plugins. If you don't find the "common-gateway/xxllnc-zgw-bundle" plugin listed here, you can search for it by clicking on "Search" in the upper-right corner and typing "xxllnc" in the search bar.
+3.  Click on the "common-gateway/xxllnc-zgw-bundle" card and then click on the "Install" button to install the plugin.
+4.  The admin-ui allows you to install, upgrade, or remove bundles. However, to load all the required data (schemas, endpoints, sources), you need to execute the initialization command in a php terminal.
+
+### XxllncZGWBundle Installation - Terminal
+
+1.  Open a php terminal and run the following command to install the XxllncZGWBundle:
+
+        docker-compose exec php composer require common-gateway/xxllnc-zgw-bundle
+
+### Initialization Command (Terminal)
+
+1.  To load all the data without any specific content (like testdata), execute the following command:
+
+        docker-compose exec php bin/console commongateway:initialize
+
+    OR
+
+    To load all the data along with specific content (like testdata), run:
+
+        docker-compose exec php bin/console commongateway:initialize -data
+
+With these steps completed, the backend setup for the XxllncZGW project should be ready to use. If you encounter any issues during the installation process, seek assistance from the development team. Happy coding!
+
+## Admin UI - Setup Instructions
+
+Once the backend (and frontend) is up and running, the HuwelijksplannerBundle can be configured. To ensure proper functionality, the sources and Security Group (Default Anonymous user) need to be modified. Other adjustments are optional.
+
+### Configuration Steps:
+
+1.  **Users**
+    *   Change the passwords of the users if necessary. It is recommended that you change the email of the admin user.
+        *   Go to `Settings` in the Admin UI.
+        *   Navigate to the `Users` tab.
+        *   Select the user and edit the password.
+
+2.  **Security Group**
+    *   Add the scopes for the Default Anonymous in the Security Group.
+        *   Go to `Settings` in the Admin UI.
+        *   Navigate to the `Security Groups` tab
+        *   Locate and select `Default Anonymous` to view its details
+        *   Add the following scopes under the `Scopes` section:
```

Comment:
This step might be possible to add to the XxllncZGWBundle instead,
By adding a SecurityGroup with these scopes and updating the Default Anonymous user to use that SecurityGroup in the installation.json by using the default reference of that user 🤔

Reply:
Could be done in a different issue

Reply:
https://conduction.atlassian.net/browse/XZ-161